site/src/pages/ManagementSettingsPage/GroupsPage/CreateGroupPageView.tsx

```diff
@@ -0,0 +1,94 @@
+import TextField from "@mui/material/TextField";
+import { type FormikTouched, useFormik } from "formik";
+import type { FC } from "react";
+import { useNavigate } from "react-router-dom";
+import * as Yup from "yup";
+import { isApiValidationError } from "api/errors";
+import type { CreateGroupRequest } from "api/typesGenerated";
+import { ErrorAlert } from "components/Alert/ErrorAlert";
+import {
+  FormFields,
+  FormFooter,
+  FormSection,
+  HorizontalForm,
+} from "components/Form/Form";
+import { IconField } from "components/IconField/IconField";
+import { PageHeader, PageHeaderTitle } from "components/PageHeader/PageHeader";
+import { getFormHelpers, onChangeTrimmed } from "utils/formUtils";
+
+const validationSchema = Yup.object({
+  name: Yup.string().required().label("Name"),
+});
+
+export type CreateGroupPageViewProps = {
+  onSubmit: (data: CreateGroupRequest) => void;
+  error?: unknown;
+  isLoading: boolean;
+  // Helpful to show field errors on Storybook
+  initialTouched?: FormikTouched<CreateGroupRequest>;
```

Comment:
We can trigger the "touching" event using the `play` feature in Storybook. Personally, I don't like having props that are only meant for testing a specific framework.